### PR TITLE
add: crystal language icon

### LINKED ISSUE #263: add Crystal Language 💎
## The icon I want is:

* Name: Crystal Language
* URL: https://crystal-lang.org
* Link to icon: https://crystal-lang.org/assets/media/crystal_icon.svg
* Official style guide or brand guide: https://crystal-lang.org/media/

### PR DESCRIPTION
**Pull request by johncurcio**
Add crystal language icon as requested in #263.

Closes #263

### CODE REVIEW

**Comment by edent** [comment]
That's a good start. Please can you follow the template in https://github.com/edent/SuperTinyIcons/blob/master/CONTRIBUTING.md - that will help make it much smaller.

**Comment by johncurcio** [comment]
I had to re-scale the svg to match the 512x512 rectangle, so the code changed a little bit.

File: images/svg/crystal.svg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<svg viewBox="0 0 512 512" xmlns="http://www.w3.org/2000/svg" aria-label="crystal">
+  <rect fill="#fff" height="512" rx="15%" width="512"/>
+  <path d="M448.231 308.479L308.661 447.77c-.559.558-1.396.838-1.955.558l-190.653-51.082c-.838-.28-1.396-.838-1.396-1.396L63.295 205.475c-.28-.837 0-1.395.558-1.954L203.424 64.23c.558-.558 1.396-.838 1.954-.558l190.654 51.082c.837.28 1.395.838 1.395 1.396l51.083 190.375c.559.837.28 1.395-.279 1.954zM261.206 156.905L73.902 206.87c-.279 0-.558.558-.279.838l137.059 136.779c.279.279.837.279.837-.28l50.246-187.024c.279 0-.28-.558-.559-.28z"/>
+  <path fill="none" d="M-458.602-458.602H970.602V970.602H-458.602z"/>
```
Comment (edent) [inline images/svg/crystal.svg]: What does this path do? Doesn't seem to make a difference to the image if it is removed. Or am I missing something?

Comment (johncurcio) [reply images/svg/crystal.svg]: You're right, it's not doing anything. I think it has something to do with the svg editor I use.

**Pull request merged**